lib/plugins/aws/configCredentials/awsConfigCredentials.js

```diff
@@ -64,19 +65,13 @@ class AwsConfigCredentials {
     this.serverless.cli.log('Setting up AWS...');
     this.serverless.cli.log('Saving your AWS profile in "~/.aws/credentials"...');
 
-    // locate home directory on user's machine
-    const env = process.env;
-    const home = env.HOME ||
-      env.USERPROFILE ||
-      (env.HOMEPATH ? ((env.HOMEDRIVE || 'C:/') + env.HOMEPATH) : null);
-
-    if (!home) {
+    if (!os.homedir()) {
```

Comment:
Yep. That totally makes sense.

I copied the code for the homedir detection from an old PR and pushed the test on top of that 😬. But yes, if the test use os.homedir and work for all node versions on our CI system, then the usage of `os.homedir` should work in the main code as well 😆 🤦‍♂️